subsys/bluetooth/host/smp.c

```diff
@@ -1239,9 +1239,14 @@ static void smp_br_distribute_keys(struct bt_smp_br *smp)
 	if (smp->local_dist & BT_SMP_DIST_SIGN) {
 		struct bt_smp_signing_info *info;
 		struct net_buf *buf;
+		uint8_t csrk[16];
 
 		smp->local_dist &= ~BT_SMP_DIST_SIGN;
 
+		if (bt_rand(csrk, sizeof(csrk))) {
+			return;
```

Comment:
Probably better to add a BT_ERR() here since otherwise you'll be scratching your head why stuff doesn't work if this happens

Reply:
It is a BT_ERR here, just added in a later commit since there was a few other places that could need BT_ERR when bt_rand failed.

Reply:
Ok!